samples/createBucketWithTurboReplication.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic operations on buckets with
+ * the Google Cloud Storage API.
+ *
+ * For more information, see the README.md under /storage and the documentation
+ * at https://cloud.google.com/storage/docs.
+ */
+
+function main(
+  bucketName = 'my-bucket',
+  location = 'NAM4',
+  rpo = 'ASYNC_TURBO'
+) {
+  // [START storage_create_bucket_turbo_replication]
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // The ID of your GCS bucket
+  // const bucketName = 'your-unique-bucket-name';
+
+  // The name of a dual region location
+  // See this documentation for other valid locations:
+  // https://cloud.google.com/storage/docs/locations#location-dr
+  // const location = 'NAM4';
+
+  // Flag to enable turbo replication for this bucket
+  // const rpo = 'ASYNC_TURBO';
+
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  // The bucket in the sample below will be created in the project associated with this client.
+  // For more information, please see https://cloud.google.com/docs/authentication/production or https://googleapis.dev/nodejs/storage/latest/Storage.html
+  const storage = new Storage();
+
+  async function createBucketWithTurboReplication() {
+    // For default values see: https://cloud.google.com/storage/docs/locations and
+    // https://cloud.google.com/storage/docs/storage-classes
+    const [bucket] = await storage.createBucket(bucketName, {
+      location,
+      rpo,
+    });
+
+    console.log(
+      `${bucket.name} created with turbo replication ${rpo} class in ${location}.`
+    );
+  }
+
+  createBucketWithTurboReplication().catch(console.error);
```

Comment:
Instead of catching like this, please remove the `.catch`, and add this at the end:

```js
process.on('unhandledRejection', err => {
  console.error(err.message);
  process.exitCode = 1;
});
main(...process.argv.slice(2));
```

This follows the sample standards we've laid out elsewhere.

Reply:
Thanks @JustinBeckwith I'll make a ticket to update the other tests to this format as most in this repo are doing `.catch(console.error)`